test/manual/test_percl_generation.py

```diff
@@ -0,0 +1,63 @@
+
+"""
+    FreeClimb API
+
+    This is a manually implemented test to validate the 
+    contents of https://github.com/FreeClimbAPI/Python-Getting-Started-Tutorial/blob/master/python-getting-started.py
+    are functioning as expected
+"""
+
+import unittest
+
+import freeclimb
+import json
+
+
+class TestQuickStart(unittest.TestCase):
+    def setUp(self):
+        pass
+
+    def tearDown(self):
+        pass
+
+    def testPerclGeneration(self):
+        message = "Hello, FreeClimb!"
+        say = freeclimb.Say(text=message)
+        getSpeech = freeclimb.GetSpeech(
+            action_url="http://example.com/actionurl/getspeech", prompts=[say], grammar_file="grammar.xml")
+        recordUtterance = freeclimb.RecordUtterance(
```

Comment:
is python meant to be in snake case or camo case for variable names? the variable names look a bit inconsistent.